sysdeps/linux/aarch64/thread_entry.S

```diff
@@ -0,0 +1,25 @@
+.section .text
+.global __mlibc_spawn_thread
+.type __mlibc_spawn_thread, "function"
+__mlibc_spawn_thread:
+	// __mlibc_spawn_thread(flags, stack, pid_out, child_tid, tls)
```

Comment:
Why are these commented out? Why can't we just remove these lines?

Reply:
These comments are explaining what we're doing with the argument registers before the clone syscall (similar to the riscv sysdeps).